backend/main.py

```
@@ -736,6 +738,8 @@ async def _chairman_stream(
 
         # Build context messages from conversation history
         conv = storage.get_conversation(conversation_id)
+        if conv is None:
+            raise HTTPException(status_code=404, detail="Conversation not found")
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)